lib/OpenQA/Utils.pm

```diff
@@ -349,20 +353,30 @@ sub run_cmd_with_log_return_error ($cmd, %args) {
             log_error("cmd returned $return_code");
         }
         return {
-            status => $ipc_run_succeeded,
+            status => $ipc_run,
             return_code => $return_code,
             stdout => $stdout,
             stderr => $stderr,
         };
     }
     catch {
-        return {
-            status => 0,
-            return_code => undef,
-            stderr => 'an internal error occurred',
-            stdout => '',
+        my $additional_info = "If you use http for CASEDIR or NEEDLES_DIR is adviced to
+        configure git with the insteadof option in the .gitconfig file.
+        Run the following command to:
+        git config [--global] url.\"git://github.com\".insteadOf https://github.com";
```

Comment:
`url.insteadOf` is **not** what we want. it is `url.pushInsteadOf`, which is an important difference.
We still want the **fetch** to happen with http.